### ISSUE TITLE
Strange aggro behavior 

### ISSUE THREAD

**Issue by Kangaroux** [slowtorta/turtlewow-bug-tracker#337]
https://database.turtle-wow.org/?npc=4030

Lvl 44 spriest (me) with a lvl 26 hunter. Hunter tagged the mob and I followed up with mind blast (not resisted), which caused the mob to start attacking me. Hunter used arcane shot shortly after and the mob switched aggro to the hunter.

Just prior to this we fought the elite harpy (https://vanillawowdb.com/?npc=12579) but didn't have any aggro issues.

Is this intended? I'm not sure how this happened. Arcane shot shouldn't be generating more threat than mind blast.

**Comment by Kangaroux** [comment]
> Sorry, this sounds like normal threat behavior.

Can you give some detail on why this is normal?

**Comment by CosminPOP** [comment]
You're right that mindblast should do more threat but the info you provided is not enough. We dont know how much damage each player did to that mob.
